cmd/sovereignnode/incomingHeader/incomingEventsProcessor.go

```diff
@@ -65,9 +66,21 @@ func (iep *incomingEventsProcessor) processIncomingEvents(events []data.EventHan
 		case topicIDDeposit:
 			scr, err = iep.createSCRInfo(topics, event)
 			scrs = append(scrs, scr)
-		case topicIDExecutedBridgeOp:
-			confirmedOp, err = iep.getConfirmedBridgeOperation(topics)
-			confirmedBridgeOps = append(confirmedBridgeOps, confirmedOp)
+		case topicIDExecuteBridgeOps:
+			if len(topics) == 0 {
+				return nil, errInvalidNumTopicsIncomingEvent
+			}
+
+			switch string(topics[0]) {
+			case topicIDDeposit:
+				scr, err = iep.createSCRInfo(topics, event)
+				scrs = append(scrs, scr)
```

Comment:
should the failures not be added to the confirmedBridgeOps?

Reply:
There are 2 kinds of `event IDs`: 
- `deposit` which is a normal deposit to the other chain
- `executeBridgeOps` - after a deposit in sovereign, the bridge service sends (execute brdige operations) the transactions to mainchain. This operation will always return event with `topicIDExecutedBridgeOp`, and another event with `topicIDDeposit` only if tx is fail (this means to return back the tokens on sovereign chain)

So, all txs (even failures) should be confirmed back to sovereign.